ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -846,6 +846,20 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
+- as: aws-ipi-ovn-efs-hypershift-mgmt-longduration-f28-destructive
+  cron: 13 18 4 * *
+  steps:
+    cluster_profile: aws-qe
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.19
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_FILTERS: NonPreRelease&;HyperShiftMGMT&;NonHyperShiftHOST&;Longduration&;Serial&;Disruptive&;~ChkUpgrade&;~MicroShiftOnly&
+      TEST_IMPORTANCE: Medium
+      TEST_TIMEOUT: "90"
+    test:
+    - chain: openshift-e2e-test-hypershift-qe-mgmt-longrun
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest-efs
```

Comment:
you should not use guest workflow in mgmt job

Reply:
Ack, thanks. My need is to install EFS operator in Guest cluster and this workflow helps to install it on guest cluster. 
Is there any alternative way which makes sure to install the operator in Guest cluster?

Reply:
If you want to test in mgmt cluster please add a new workflow e.g. `cucushift-installer-rehearse-aws-ipi-ovn-hypershift-efs` after install the efs csi driver switch default kubeconfig back to mgmt cluster kubeconfig by using `ref:cucushift-hypershift-extended-disable-guest`.